File: src/main/java/com/ulaf/ste/ordering_system/Web/requests/OrderRequest.java
package com.ulaf.ste.ordering_system.Web.requests;

import com.ulaf.ste.ordering_system.Model.OrderItem;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.GenerationType;
import jakarta.persistence.Id;
import lombok.*;

import java.util.Date;
import java.util.List;
@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
@Data
public class OrderRequest {
     List<OrderItemRequest> orderItemRequests;
     String customerName;
     String customerSurname;
     String customerEmailAddress;
     String customerPhone;
     String deliveryAddress;

}

File: frontend/src/model/OrderRequest.ts
import {OrderItem} from "./OrderItem";
import {OrderItemRequest} from "./OrderItemRequest";

export interface OrderRequest {
  orderItemRequests: OrderItemRequest[],
  customerName:String,
  customerSurname:String,
  customerEmailAddress:String,
  customerPhone:String,
  deliveryAddress: String,
  storePickup: boolean,
}
